recipes/sfml/3.x/conanfile.py

```diff
@@ -0,0 +1,250 @@
+
+        modules = ["system"]
+        if self.settings.os in ["Windows", "iOS", "Android"]:
+            modules.append("main")
+
+        modules.extend(module for module in ["window", "graphics", "network", "audio"] if self.options.get_safe(module))
+
+        for module in modules:
+            self._basic_module_definition(module)
+
+        # System module is always required
+        if self.settings.os in ["Linux", "FreeBSD"]:
+            self.cpp_info.components["system"].system_libs = ["pthread", "rt"]
+        elif self.settings.os == "Windows":
+            self.cpp_info.components["system"].system_libs = ["winmm"]
+        elif self.settings.os == "Android":
+            self.cpp_info.components["system"].system_libs = ["log", "android"]
+
+        if self.options.window:
+            self.cpp_info.components["window"].requires = ["system", "vulkan-headers::vulkan-headers"]
+            if self.settings.os in ["Linux", "FreeBSD"]:
+                self.cpp_info.components["window"].system_libs.append("dl")
+                self.cpp_info.components["window"].requires.extend(["xorg::x11", "xorg::xrandr", "xorg::xcursor", "xorg::xi"])
+
+            if self.settings.os == "iOS":
+                self.cpp_info.components["window"].frameworks = ["OpenGLES"]
+            elif self.settings.os == "Android":
+                self.cpp_info.components["window"].system_libs.extend(["egl", "GLESv2"])
+            else:
+                self.cpp_info.components["window"].requires.append("opengl::opengl")
+
+            if self.settings.os == "Linux":
+                self.cpp_info.components["window"].requires.append("libudev::libudev")
+            elif self.settings.os == "Windows":
+                self.cpp_info.components["window"].system_libs.extend(["gdi32", "winmm"])
+            elif self.settings.os == "FreeBSD":
+                self.cpp_info.components["window"].system_libs.append("usbhid")
+            elif self.settings.os == "Macos":
+                # CoreServices is pulled from Carbon, even if it does not show up in the upstream CMakeLists.txt
+                self.cpp_info.components["window"].frameworks = ["Foundation", "AppKit", "IOKit", "Carbon", "CoreServices"]
+            elif self.settings.os == "iOS":
+                self.cpp_info.components["window"].frameworks = ["Foundation", "UIKit", "CoreGraphics", "QuartzCore", "CoreMotion"]
+            elif self.settings.os == "Android":
+                self.cpp_info.components["window"].system_libs = ["android"]
+
+        if self.options.graphics:
+            self.cpp_info.components["graphics"].requires = ["window", "stb::stb"]
+            if self.settings.os == "Android" or self.settings.os == "iOS":
+                self.cpp_info.components["graphics"].requires.append("zlib::zlib")
+            if self.settings.os == "iOS":
+                self.cpp_info.components["graphics"].requires.append("bzip2::bzip2")
```

Comment:
https://github.com/SFML/SFML/blob/3.0.2/src/SFML/Graphics/CMakeLists.txt#L105
```suggestion
                self.cpp_info.components["graphics"].requires.extend(["zlib::zlib", "bzip2::bzip2"])
```

Reply:
Yeah, my fault 😅